src/printer_internal.h

```diff
@@ -29,6 +29,14 @@ typedef enum LYOUT_TYPE {
     LYOUT_CALLBACK     /**< print via provided callback */
 } LYOUT_TYPE;
 
+/**
+ * @brief Types of the YANG printers
+ */
+enum schema_type {
+    YPR_PARSED,   /**< YANG printer of the parsed schema */
+    YPR_COMPILED  /**< YANG printer of the compiled schema */
+};
+
```

Comment:
I think that now, when the compiled YIN printer was removed, this change should be reverted and the enum should be moved back into printer_yang.c as it was.

Reply:
OK, I will revert this.